frontend/packages/process-editor/src/components/Canvas/CanvasActionMenu/CanvasActionMenu.tsx

```diff
@@ -21,33 +18,21 @@ export type CanvasActionMenuProps = {
  *
  * @returns {ReactNode} - The rendered component
  */
-export const CanvasActionMenu = ({
-  isEditorView,
-  onSave,
-  toggleViewModus,
-}: CanvasActionMenuProps): ReactNode => {
+export const CanvasActionMenu = ({ onSave }: CanvasActionMenuProps): ReactNode => {
   const { t } = useTranslation();
   const { numberOfUnsavedChanges } = useBpmnContext();
 
   return (
     <span className={classes.canvasMenuContainer}>
-      <Button
-        onClick={toggleViewModus}
-        variant='secondary'
-        icon={isEditorView ? <EyeFillIcon /> : <CogIcon />}
-      >
-        {isEditorView ? t('process_editor.view_mode') : t('process_editor.edit_mode')}
-      </Button>
       {numberOfUnsavedChanges > 0 && (
         <span className={classes.unsavedChanges}>
           {t('process_editor.unsaved_changes', { count: numberOfUnsavedChanges })}
         </span>
       )}
-      {isEditorView && (
-        <Button onClick={onSave} color='success'>
-          {t('process_editor.save')}
-        </Button>
-      )}
+
+      <Button className={classes.saveButton} onClick={onSave} color='success'>
```

Comment:
The button should only be displayed if the user has the correct version and is allowed to edit the bpmn, I'll suggest this: 
```tsx
{isEditAllowed && (
        <Button onClick={onSave} color='success'>
          {t('process_editor.save')}
        </Button>
}
```